packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -464,7 +415,7 @@ export class PerformanceEvent {
     protected constructor(
         private readonly logger: ITelemetryLogger,
         event: ITelemetryGenericEvent,
-        private readonly markers: IPerformanceEventMarkers = {start: true, end: true, cancel: "generic"},
```

Comment:
See https://github.com/microsoft/FluidFramework/pull/5897/files that goes into more details, preserve start in couple places, etc.